pkg/atomicbitops/atomicbitops_amd64.s

```diff
@@ -16,21 +16,21 @@
 
 #include "textflag.h"
 
-TEXT ·AndUint32(SB),$0-12
+TEXT ·AndUint32(SB),$8-12
   MOVQ  addr+0(FP), BP
```

Comment:
Why do these functions use BP at all? I'm pretty sure this is just a typo for BX, and we should just change to BX.

Reply:
I see. I'll fix BP to BX at all.